README.md

```diff
@@ -452,12 +488,25 @@ The output is a list of statistics for each question from the reference Q&A data
 - `answer_relevance_error`: (optional) error message if answer relevance evaluation failed
 - `answer_relevance_cost`: The LLM use cost of computing `answer_relevance`, in US dollars
 - `actual_steps`: (optional) copy of the steps in the evaluation target, if specified there
-- `steps_score`: a real number between 0 and 1, computed by comparing the results of the last steps that were executed to the reference's last group of steps. If there is no match in the actual steps, then the score is `0`. Otherwise, it is calculated as the number of the matched steps on the last group divided by the total number of steps in the last group.
+- `steps_score`: a real number between 0 and 1, computed by comparing the results of the last executed steps to the output of the reference's last group of steps.
+    - If there is no match in the actual steps, then the score is `0.0`
+    - For executed steps whose name is "retrieval":
+        - If the last reference group contains a retrieval step and its output (the reference context) is specified, the score is the [recall at k](#context-recallk) of the retrieved document ids with respect to the reference.
+        - If it does not specify the output (there is no reference context), the score is `1.0`
+    - Otherwise, the score is the number of the matched steps on the last group divided by the total number of steps in the last group.
 - `input_tokens`: input tokens usage
 - `output_tokens`: output tokens usage
 - `total_tokens`: total tokens usage
 - `elapsed_sec`: elapsed seconds
 
+All `actual_steps` with `name` "retrieval" contain:
+- `retrieval_answer_recall`: recall of the actually retrieved context when evaluated using the reference answer
+- `retrieval_answer_recall_cost`: cost of evaluating `retrieval_answer_recall`, in US dollars
+- `retrieval_answer_precision`: precision of the actually retrieved context when evaluated using the reference answer
+- `retrieval_answer_precision_cost`: cost of evaluating `retrieval_answer_precision`, in US dollars
+- `retrieval_answer_f1`: F1 score of the actually retrieved context when evaluated using the reference answer
+- `retrieval_answer_f1_cost`: cost of evaluating `retrieval_answer_f1`, in US dollars
```

Comment:
Should we clarify that this is the sum of retrieval_answer_recall_cost and retrieval_answer_precision_cost?